todo-app/src/TodoItem.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { observer } from "mobx-react";
+import Description from './Description';
+import Deadline from './Deadline';
+import './index.css';
+
+
+@observer
```

Comment:
Is it necessary to make the `TodoItem` component a MobX-observer? `TodoItem` is not getting its data directly from the store, only from its props, so you don't need to do this.

Reply:
I think I need that, 'deleteTodo' and 'changeStatus' do not functioning otherwise.

Reply:
See #38 and Ji's [detailed explanation here](https://hackyourfuture.slack.com/archives/C6CK8SF7S/p1516371508000559).